python/README.md

```diff
@@ -50,21 +50,39 @@ Installation
 4) Build and run the tests:
 
      $ python setup.py build
-     $ python setup.py google_test
+     $ python setup.py test
 
-     To build the C++ implementation run:
-     $ python setup.py build --cpp_implementation
+     To build, test, and use the C++ implementation, you must first compile
+     libprotobuf.so:
+
+     $ (cd .. && make)
+
+     On OS X:
```

Comment:
I think DYLD_LIBRARY_PATH on OS X behaves like LD_LIBRARY_PATH on Linux? http://stackoverflow.com/questions/3146274/is-it-ok-to-use-dyld-library-path-on-mac-os-x-and-whats-the-dynamic-library-s